src/python/pants/backend/jvm/subsystems/compile_subsystem.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+from pants.subsystem.subsystem import Subsystem
+
+
+logger = logging.getLogger(__name__)
+
+
+class JvmCompileSubsystem(Subsystem):
```

Comment:
It seems like this option could probably go on `src/python/pants/backend/jvm/subsystems/jvm_platform.py` instead as `--jvm-platform-compiler`?